### PR TITLE
Use GitHub Actions for CI

### PR DESCRIPTION
**Pull request by garethbowen**
Replace travis-ci integration with GitHub Actions.

#138 
#129 
#91

### CODE REVIEW

**Comment by garethbowen** [comment]
@mrsarm Please review when you get a chance. Hopefully this will make it easier to release the permissions changes you're working on.

I haven't solved publishing to Play Store, and I'm thinking about splitting that off as a separate issue because I'm not sure it's a good idea anyway, but this PR will unblock App Builders by publishing apks with the GH Release.

**Review by mrsarm — APPROVED** [review]
Nothing to add. Looks great!

Though I see in the build of this PR that GH is waiting for travis despite you did remove the travis configuration file, but I guess it is because there is some configuration in the repo that you will remove before the PR merge.

**Comment by garethbowen** [comment]
@mrsarm Thanks. I've merged this and changed the repo rules so it requires the GH build not the travis build. This means you'll have to rebase your branch to get it to trigger the GH build. Hopefully this speeds up your CI!

**Pull request merged**
